src/app/contexts/ToggleContext/utils/constructTogglesEndpoint/index.js

```diff
@@ -0,0 +1,15 @@
+import onClient from '#lib/utilities/onClient';
+
+const getTogglesEndpoint = (service, origin) => {
+  const requestOrigin = origin || 'https://www.test.bbc.com';
+  const baseTogglesUrl = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`;
+
+  // client side renders should trigger a geoIPLookup
+  if (onClient()) {
+    return `${baseTogglesUrl}&geoiplookup=true`;
+  }
+
+  return baseTogglesUrl;
+};
+
+export default getTogglesEndpoint;
```

Comment:
Can we just add `export default` in front of the function above? It currently does not match the file name.